rules/linux/initial_access_login_failures.toml

```
@@ -0,0 +1,51 @@
+[metadata]
+creation_date = "2020/07/08"
+ecs_version = ["1.5.0"]
+maturity = "development"
+updated_date = "2020/07/08"
+
+[rule]
+author = ["Elastic"]
+description = "Identifies that the maximum number of failed login attempts has been reached for a user."
+index = ["auditbeat-*"]
+interval = "10m"
+language = "kuery"
+license = "Elastic License"
+name = "Auditd Max Failed Login Attempts"
+references = ["https://github.com/linux-pam/linux-pam/blob/0adbaeb273da1d45213134aa271e95987103281c/modules/pam_faillock/pam_faillock.c#L574"]
+risk_score = 47
+rule_id = "fb9937ce-7e21-46bf-831d-1ad96eac674d"
+severity = "medium"
+tags = ["Elastic","Linux"]
+type = "query"
+
+query = '''
+auditd.message_type:anom_login_failures and event.module:auditd
```

I would flip all of these. In general, it's always felt easier for me to read logic that starts with categorization and gets more specific the more you read.
For reference https://github.com/elastic/detection-rules/blob/main/CONTRIBUTING.md#using-elastic-common-schema-ecs
```suggestion
event.module:auditd and auditd.message_type:anom_login_failures 
```

---

Done